### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #590 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#348]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 4.86 KB...]

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running smartcity.accessibility.mapmanagement.jxMapsFunctionalityTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.066 sec - in smartcity.accessibility.mapmanagement.jxMapsFunctionalityTest
Running smartcity.accessibility.mapmanagement.LocationTest

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00007f91457b0000, 65536, 1) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 65536 bytes for committing reserved memory.
[thread 140262295348992 also had an error]
# An error report file with more information is saved as:
# /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/hs_err_pid6307.log
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000001002e0000, 131072, 0) failed; error='Cannot allocate memory' (errno=12)
#
# Compiler replay data is saved as:
# /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/replay_pid6307.log

Results :

Tests run: 1, Failures: 0, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:28 min
[INFO] Finished at: 2017-05-17T08:09:41+00:00
[INFO] Final Memory: 33M/157M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/target/jacoco.exec -jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/target/surefire/surefirebooter534814470966988901.jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/target/surefire/surefire3469897453694476480tmp /var/lib/jenkins/sharedspace/SmartCity-Accessibility@2/server-side/target/surefire/surefire_04831590253957925780tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Arthur Sapozhnikov] de4f0bb4ded1996374e473969f8a2fcfeee9cb4d - Minor change in initial rating

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/590/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
